src/lib/render.jsx

```diff
@@ -35,9 +37,21 @@ const render = (jsx, element, reducers) => {
     }
 
     const allReducers = reducer(reducers);
+    
+    const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || redux.compose;
+    const enhancers = enhancer ?
+        composeEnhancers(
+            redux.applyMiddleware(thunk),
+            enhancer
+        ) :
+        composeEnhancers(
+            redux.applyMiddleware(thunk)
+        );
+    const initState = preload || {};
     const store = redux.createStore(
         allReducers,
-        redux.applyMiddleware(thunk)
+        initState,
```

Comment:
If this came in as `initialState`, then the above assignment could be inlined as `initialState || {}` here.